docs/broker.rst

```diff
@@ -88,6 +88,73 @@ its secret type updates to the secret object in the Kubernetes API will be autom
 a Pod's filesystem. Hpfeeds will spot those updates and process them immediately without needing a
 restart.
 
+Database authentication store
+--------------------------
+
+When starting the broker you can pass a database connection string. Auth requests are then checked against
+the selected Database in a table named auth_keys. 
+
+Supported Database drivers are:
+  - postrgesql
+  - mysql
+  - mysql compatable e.g. mariadb, aurora
+  - sqlite
+
+You may need to install the specific database driver for your selected database, more information can be 
+found at https://pypi.org/project/databases/
+
+.. code-block: bash
+
+  $ pip install databases[postgresql]
+  $ pip install databases[mysql]
+  $ pip install databases[sqlite]
+
+Using SQLite with this auth mechanism requires JSON support that can be found in SQLite version > 3.3 and Python3
+Previous versions of SQLite may be supported with the JSON1 SQLite extension.
+
+Any authentication can be included within the connection string
+For example:
+
+.. code-block:: bash
+
+    hpfeeds-broker -e tcp:port=20000 --exporter=0.0.0.0:9431 --auth="database+mysql://honeyswarm:honeyswarm@127.0.0.1/honeyswarm"
```

Comment:
Would be super happy to link to honeyswarm from the docs (or the main site -> https://github.com/hpfeeds/hpfeeds.github.io), but maybe we should stick to `username:password@127.0.0.1/example` here?

Reply:
Sorry, that was a lazy copy and paste :)